x-pack/plugins/cloud_security_posture/public/pages/findings/findings_flyout.tsx

```diff
@@ -21,12 +21,15 @@ import {
   EuiFlexGrid,
   EuiCard,
   EuiFlexGroup,
+  EuiIcon,
   type PropsOf,
 } from '@elastic/eui';
 import { assertNever } from '@kbn/std';
 import type { CspFinding } from './types';
 import { CspEvaluationBadge } from '../../components/csp_evaluation_badge';
 import * as TEXT from './translations';
+import cisLogoIcon from '../../assets/icons/cis_logo.svg';
+import k8sLogoIcon from '../../assets/icons/k8s_logo.svg';
```

Comment:
the figma design includes `png` for both icons, but they're common so i used `svg` i found

this also works well with `EuiIcon` which accepts custom `svgs`

if this is a mistake, LMK